app/src/main/java/fr/free/nrw/commons/category/CategoryImagesListFragment.java

```diff
@@ -183,6 +183,7 @@ private void setScrollListener() {
         gridView.setOnScrollListener(new AbsListView.OnScrollListener() {
             @Override
             public void onScrollStateChanged(AbsListView view, int scrollState) {
+                view.setFastScrollEnabled(scrollState!=SCROLL_STATE_IDLE);
```

Comment:
formatting needs correcting, there should be spaces around operators

Reply:
will do that.

Reply:
This works fine if you drag scrollbar while scrolling and completely eliminates abrupt jumps but for idle state you can't drag it.
Also hiding it doesn't work and still manages to activate on accidental touch.

Reply:
Did you test it on device to see if it works fine. 
Although, i came up with one more solution but it will require lot more code.
Firstly change the width(styling) of scrollbar and then set an offset(this can be width of scrollbar) such that fastscroll is enabled only when user scrolls within that offset. This reduces abrupt jumps to great extent.

Reply:
I was just validating code quality. I don't think that reading the issue there is much consensus on what is to be done. Personally I'd be more for deleting fast scroll, to me flinging is very quick at getting through data and I don't understand why people want to be scrolling through `explore>uploaded via mobile` arbitrarily? Are people using it as a `Random` feature essentially? Is it a most recent list and if so why do they want to see the 1000th most recent item? I just don't understand who the feature is for so I can't make a recommendation on what to actually do.

@maskaravivek has been leading the charge on this ticket so I would seek his review

Reply:
I tend to agree with @macgills's comments and am not sure if fast scrolling truly adds any value or not. Lets discuss it in the issue thread.